{
  "title": "Get Training Status of LargePersonGroup",
  "operationId": "PersonGroupOperations_GetLargePersonGroupTrainingStatus",
  "parameters": {
    "apiVersion": "v1.1-preview.1",
    "largePersonGroupId": "your_large_person_group_id"
  },
  "responses": {
    "200": {
      "body": {
        "status": "notStarted",
        "createdDateTime": "2024-03-05T11:07:58.371Z",
        "lastActionDateTime": "2024-03-05T11:07:58.371Z",
        "lastSuccessfulTrainingDateTime": "2024-03-05T11:07:58.371Z",
        "message": null
      }
    },
    "401": {
      "body": {
        "error": {
          "code": "Unspecified",
          "message": "Access denied due to invalid subscription key. Make sure you are subscribed to an API you are trying to call and provide the right key."
        }
      }
    },
    "403": {
      "body": {
        "error": {
          "statusCode": 403,
          "message": "Out of call volume quota. Quota will be replenished in 2 days."
        }
      }
    },
    "404": {
      "body": {
        "error": {
          "code": "LargePersonGroupNotFound",
          "message": "Large person group is not found."
        }
      }
    },
    "409": {
      "body": {
        "error": {
          "code": "ConcurrentOperationConflict",
          "message": "There is a conflict operation on requested resource, please try later."
        }
      }
    },
    "429": {
      "body": {
        "error": {
          "statusCode": 429,
          "message": "Rate limit is exceeded. Try again in 26 seconds."
        }
      }
    }
  }
}
